### ISSUE TITLE
Build adjustment for aquaterm-1.1.1 on Mojave

### ISSUE THREAD

**Issue by akhansen** [fink/fink-distributions#352]
Dear Mr. Kevin Horton,

Thank you very much for maintaining AquaTerm in Fink.
Today,  I have built aquaterm 1.1.1-4 in fink 0.44.0
on my Mac running over MacOS 10.14.3 (Mojave).
I have found that a small modification for aquaterm.info is necessary:
`--------------------------------------------------------------------------------------------`
`--- aquaterm.info.org 2018-03-05 20:30:12.000000000 +0900`
`+++ aquaterm.info 2019-02-13 09:18:12.000000000 +0900`
`@@ -13,10 +13,12 @@`
` CompileScript: echo nothing to do here, everything is done in the installscript`
` InstallScript: <<`
`  #!/bin/sh -ex`
`+ . %p/sbin/fink-buildenv-helper.sh`
`+ if test "$XCODE_MAJOR_VERSION" -ge "10"; then xcodebuildoption="-UseNewBuildSystem=NO";` `else xcodebuildoption=""; fi`
`  if test "%m" = "powerpc"; then archname="ppc"; else archname="%m"; fi`
`  ARCHS=$archname`
`  cd aquaterm`
`- xcodebuild install -target AquaTerm DSTROOT=%d INSTALL_PATH=%p/Applications DYLIB_INSTALL_NAME_BASE=%p/Library/Frameworks GCC_PREPROCESSOR_DEFINITIONS="AQT_APP" ARCHS=$archname MACOSX_DEPLOYMENT_TARGET=10.7`
`+ xcodebuild install $xcodebuildoption -target AquaTerm DSTROOT=%d INSTALL_PATH=%p/Applications DYLIB_INSTALL_NAME_BASE=%p/Library/Frameworks GCC_PREPROCESSOR_DEFINITIONS="AQT_APP" ARCHS=$archname MACOSX_DEPLOYMENT_TARGET=10.7`
`  mkdir -p %i/include/aquaterm`
`  cp AQTAdapter.h aquaterm.h %i/include`
`  cp AQTAdapter.h %i/include/aquaterm`
`------------------------------------------------------------------------------------------------------------------`
Namely, an additional option  "`-UseNewBuildSystem=NO`"  is necessary to be
added to xcodebuild on Mojave. 
Please use this information for your maintenance of AquaTerm.
Again, thank you very much for your effort to help huge users of fink.

Satoshi Adachi

**Comment by akhansen** [comment]
Also, we need to find a new maintainer or change it to None.

**Comment by nieder** [comment]
Theres's some discussion on this in #334 .

**Comment by sth0** [comment]
Another problem is that there is no source for the build and thus it fails under 10.14.  The original source is long gone and the "new" aqua term site has a different file name with different MD5 hash.
It is not clear that the "new" site is active at all but at least has a source copy.

Without aquaterm a lot of other things fail such as gnuplot, ghostscript.

**Comment by nieder** [comment]
````
$ diff -ruN AquaTerm-AquaTerm-5c223a5 AquaTerm-1.1.1
[Bubble:/src] nieder $
````
`AquaTerm-AquaTerm-5c223a5` is from my previously downloaded file when the mirrors were still working. `AquaTerm-1.1.1` is from <https://sourceforge.net/projects/aquaterm/>.
Diffing the output of `tar -tvf <tarball>` only differs in the name of the root directory.

I've uploaded the original tarball I had downloaded from some time ago to our SF space and updated the .info to know about the new location. So that at least takes care of the missing file.

**Comment by sth0** [comment]
> On Mar 8, 2019, at 10:16 PM, nieder <notifications@github.com> wrote:
> 
> $ diff -ruN AquaTerm-AquaTerm-5c223a5 AquaTerm-1.1.1
> [Bubble:/src] nieder $
> AquaTerm-AquaTerm-5c223a5 is from my previously downloaded file when the mirrors were still working. AquaTerm-1.1.1 is from https://sourceforge.net/projects/aquaterm/ <https://sourceforge.net/projects/aquaterm/>.
> Diffing the output of tar -tvf <tarball> only differs in the name of the root directory.
> 
> I've uploaded the original tarball I had downloaded from some time ago to our SF space and updated the .info to know about the new location. So that at least takes care of the missing file
> 
Thanks.  Now I get an error building since it is trying to put a bunch of configuration files into /var/empty
2019-03-08 22:48:39.714 xcodebuild[25254:3551549] [MT] IDELogStore: Failed to open log store at /var/empty/Library/Developer/Xcode/DerivedData/AquaTerm-fckaocddlsfmfccyqadujeqgwmsz/Logs/Build

Obviously /var/empty should stay empty for a whole host of reasons.  But that is the home directory of fink-bld so there is an issue.

If I ask it to --no-build-as-nobody then I get a different set of errors…  I will take a deeper look some other time.

-Scott

**Comment by nieder** [comment]
Should be fixed with commit 3614892
